config.yaml

```diff
@@ -94,6 +94,7 @@ datasets:
         genename: hgnc_symbol
         ensg: ensembl_gene_id
       destination: *dest
+      gx_enabled: true
```

Comment:
I'm curious: The 3 proteomics files have the exact same file format, same columns, etc. The only difference should be the tissues will have different values. Do you intend to use 3 different gx scripts, or one gx script for all 3?

Reply:
My plan was to use three different scripts despite the similarities. The reason is that we need to have one GX suite per dataset and we have been using one Jupyter notebook per GX suite as well to allow for changes to be made easily. So having separate scripts will continue the pattern we have been following.